<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="CHNMCT326196L" name="PAO.PlaceAnOrderFlow" tests="1" failures="0" timestamp="08 Nov 2020 17:31:28 GMT" time="20.746" errors="1">
  <testcase name="startAnOrder" time="20.746" classname="PAO.PlaceAnOrderFlow">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at utils.DBConn.OSAUser(DBConn.java:470)
at PAO.PlaceAnOrderFlow.startAnOrder(PlaceAnOrderFlow.java:80)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:813)
]]>
    </error>
  </testcase> <!-- startAnOrder -->
</testsuite> <!-- PAO.PlaceAnOrderFlow -->
